src/docs/development/ui/advanced/slivers.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Slivers
+---
+
+A sliver is a portion of a scrollable area. You can use slivers to
+achieve custom scrolling effects.
+
+For information on implementing slivers in Flutter, including the
+[SliverAppBar](https://docs.flutter.io/flutter/material/SliverAppBar-class.html),
```

Comment:
Also add links to [SliverGrid](https://docs.flutter.io/flutter/widgets/SliverGrid-class.html) and [SliverList](https://docs.flutter.io/flutter/widgets/SliverList-class.html) here too.